src/NovelRT/CMakeLists.txt

```diff
@@ -169,3 +171,10 @@ if(WIN32)
   file(GLOB libs RELATIVE ${CMAKE_BINARY_DIR}/thirdparty ${CMAKE_BINARY_DIR}/thirdparty/*.dll)
   set_target_properties(Engine PROPERTIES RESOURCE "${libs}")
 endif()
+if(APPLE)
+  add_custom_command(TARGET Engine POST_BUILD COMMAND ${CMAKE_COMMAND} -E copy_directory
+          ${CMAKE_BINARY_DIR}/thirdparty
+          $<TARGET_FILE_DIR:Engine>)
```

Comment:
Can't this use copy_build_products instead?

Reply:
Not that I'm aware of - if so, I'd propose a separate PR for it considering it's an existing issue at that point, not something introduced by macOS support :)

Reply:
Just adding another thought - we cannot do this as we don't have a proper target to attach the DLLs/dylibs to (they get attached to Engine after being initially copied so its accessible to anything dependent on Engine)